rng.h

```diff
@@ -0,0 +1,32 @@
+#include <random>
+
+class Random
+{
+public:
+    bool BinaryEvent(float probability = 0)
+    {
+        return ApplyBernoulli(probability);
+    };
+    float AdjustDamage(int damage);
+
+private:
+    float criticalMultiplier = 1.5;
+    float criticalProbability = 1.0F / 2;
+    float damageLowerCase = 0.85;
+    float damageUpperCase = 1.05;
```

Comment:
If these are meant to be values that don't change while the program is running, they should be made const
```suggestion
    const float criticalMultiplier = 1.5;
    const float criticalProbability = 1.0F / 2;
    const float damageLowerCase = 0.85;
    const float damageUpperCase = 1.05;
```

Reply:
ah yeah should've known there was a `const` type. thanks